openedx/core/djangoapps/theming/management/commands/create_sites_and_configurations.py

```diff
@@ -205,16 +209,21 @@ def handle(self, *args, **options):
             self.discovery_base_url_fmt = "http://discovery-{site_domain}:18381/"
             self.ecommerce_oidc_url = "http://ecommerce-{}.e2e.devstack:18130/complete/edx-oidc/".format(self.dns_name)
             self.ecommerce_base_url_fmt = "http://ecommerce-{site_domain}:18130/"
+            self.journals_oidc_url = "http://journals-{}.e2e.devstack:18606/complete/edx-oidc/".format(self.dns_name)
+            self.journals_base_url_fmt = "http://journals-{site_domain}:18606/"
         else:
             configuration_prefix = "sandbox"
             self.discovery_oidc_url = "https://discovery-{}.sandbox.edx.org/complete/edx-oidc/".format(self.dns_name)
             self.discovery_base_url_fmt = "https://discovery-{site_domain}/"
             self.ecommerce_oidc_url = "https://ecommerce-{}.sandbox.edx.org/complete/edx-oidc/".format(self.dns_name)
             self.ecommerce_base_url_fmt = "https://ecommerce-{site_domain}/"
+            self.journals_oidc_url = "https://journals-{}.sandbox.edx.org/complete/edx-oidc/".format(self.dns_name)
+            self.journals_base_url_fmt = "https://journals-{site_domain}/"
 
         self.configuration_filename = '{}_configuration.json'.format(configuration_prefix)
         self.discovery_user = self.get_or_create_service_user("lms_catalog_service_user")
         self.ecommerce_user = self.get_or_create_service_user("ecommerce_worker")
+        self.journals_user = self.get_or_create_service_user("journals_worker")
```

Comment:
We should probably put this creation call behind a is_journals_enabled check like we did elsewhere.